pkg/cmd/root/root.go

```diff
@@ -98,6 +98,9 @@ func NewCmdRoot(f *cmdutil.Factory, version, buildDate string) *cobra.Command {
 		return &cmdutil.FlagError{Err: err}
 	})
 
+	// TODO does this make sense? I'd like people to be able to see usage without an auth message.
```

Comment:
Hmm that's a great point. We likely don't want an auth check on commands such as `gh`, `gh pr`, `gh pr create --help`, `gh help ...`, and similar

Reply:
I'm okay with `gh pr create --help` not working for now; there's always `gh help` and that specific help page isn't going to help much without auth (unlike the root help)